<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graf_min</class>
 <widget class="QWidget" name="Graf_min">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget_graf">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_from">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Way From  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_from">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_to">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>  To  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_to">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_result_graf_min_2">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_result_graf_min">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_check_graf_min">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit_graf_min">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_error_graf_min">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string> Login:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
